Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'govuk_markdown'
 
 # Linting
 gem 'rubocop-govuk'
+gem 'erb_lint'
```

Comment:
What does `require: false` do? It’s recommended in the ERB Lint docs, and I note that this was in place when we had ERB Lint in place before, but maybe it’s unnecessary?

Reply:
Using `require: false` on gems means those gems will not be loaded until the first time where they’re explicitly required. This saves the overhead on app startup with them being loaded every time otherwise.

Reply:
Awesome, thanks!